subcommands/version/updates.go

```diff
@@ -0,0 +1,145 @@
+package version
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"runtime"
+
+	"github.com/hashicorp/go-version"
+	tuf "github.com/theupdateframework/go-tuf/client"
+	"github.com/theupdateframework/go-tuf/data"
+)
+
+type FioctlUpdateFinder struct {
+	tuf        *tuf.Client
+	localStore *jsonFilesStore
+	platform   string
+}
+
+type FioctlUpdate struct {
+	Version *version.Version
+	Uri     string
+	Sha512  data.HexBytes
+	len     int64
+}
+
+func NewFioctlUpdateFinder(localJsonPath string) (*FioctlUpdateFinder, error) {
```

Comment:
IMHO there is no need to prefix this with `Fioctl`. Everything in this repo is `Fioctl`.

Reply:
yes, but most people see "Update" in relation to our product and think we are talking about device updates